packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -81,6 +92,8 @@ export const useNotifiSubscribe: () => Readonly<{
       const isEmailConfirmed =
         targetGroup?.emailTargets[0]?.isConfirmed ?? null;
       setIsEmailConfirmed(isEmailConfirmed);
+
+      setEmailId(targetGroup?.emailTargets[0].id ?? '');
       setEmail(email ?? '');
```

Comment:
at some point we should probably just have a `setEmailTarget` or something

Generally, having individual fields like this introduces fragility (e.g. we can have `isEmailConfirmed === false` and also `emailId === ''`, which is kind of invalid state)